derive_builder/src/options/mod.rs

```diff
@@ -198,6 +206,9 @@ impl<Mode> OptionsBuilder<Mode> where
                 // setter implicitly enabled
                 self.setter_enabled(true)
             },
+            "try_setter" => {
+                self.try_setter_enabled(true)
```

Comment:
ok, please forget that comment about implicitly enabling the standard setter - as we decided _not_ to piggy back. :-)